spec/board_spec.rb

```diff
@@ -0,0 +1,79 @@
+require './lib/board.rb'
+require './lib/player.rb'
+
+RSpec.describe Board do
+  describe "#draw?" do
+    it "check if the game is a draw" do
+    player1 =Player.new('❌')
+    player2 =Player.new('⭕️')
```

Comment:
instead of initiating it locally again, kindly use `let`

Reply:
Hi inhaq! So we cannot call a method inside a test that is not supossed to call that method? For example, we cannot call make_play on #draw? test and whats the best approach to solve this?

Reply:
Hi @noincomedev, good question. You can use hard-coded input for the function to see what output will be. That's why it's not needed to call for make_play function again and again. I hope it makes sense.